packages/shared/src/components/feeds/FeedContainer.tsx

```diff
@@ -39,6 +41,7 @@ export interface FeedContainerProps {
   showSearch?: boolean;
   besideSearch?: ReactNode;
   actionButtons?: ReactNode;
+  feedName?: AllFeedPages;
```

Comment:
You might not needs this, you can use `ActiveFeedContext`, add `feetName` there and use it without prop drilling.